lib/scss/components/news-node/_news-node.scss

```diff
@@ -93,8 +108,123 @@
   .news-social-media {
     @include small-paragraph;
 
+    a {
```

Comment:
@jenbreese 

There are duplicated selectors in this nesting. Multiple `a` and `i` selectors in the same nesting are not allowed. I'll revise in another PR. This is just FYI.